test/integration.js

```diff
@@ -294,6 +294,20 @@ describe('c8', () => {
     })
   })
 
+  describe('/* c8 ignore start/stop */', () => {
+    it('ignores lines with special comment', () => {
+      const { output } = spawnSync(nodePath, [
+        c8Path,
+        '--exclude="test/*.js"',
+        '--clean=false',
+        '--temp-directory=tmp/normal',
```

Comment:
@connorjclark part of the problem was here, we should use a different tmp directory for each test (it was an oversight that we were already reusing `tmp/normal`).

I believe this was contributing to nondeterministic behavior.